docs/docusaurus.config.js

```diff
@@ -109,14 +101,15 @@ module.exports = {
         },
       ],
     },
-    footer: {
-      copyright: `Copyright © ${new Date().getFullYear()} Rasa Technologies GmbH`,
-    },
+    // footer: {
+    //   copyright: `Copyright © ${new Date().getFullYear()} Rasa Technologies GmbH`,
+    // },
```

Comment:
Should we just remove this altogether if its being sourced by the theme?

Reply:
Yeah I might revert to this for the launch but kill it later